site/_docs/reference.md

```diff
@@ -2739,6 +2739,9 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b | FORMAT_TIMESTAMP(string timestamp)             | Formats *timestamp* according to the specified format *string*
 | s | GETBIT(value, position)                        | Equivalent to `BIT_GET(value, position)`
 | b o | GREATEST(expr [, expr ]*)                    | Returns the greatest of the expressions
+| h s | HEX(binary)                                  | Converts *binary* into a hexadecimal varchar
+| h s | HEX(bigint)                                  | Converts *bigint* into a shortened hexadecimal varchar
+| h s | HEX(varchar)                                 | Converts *varchar* into a hexadecimal varchar
```

Comment:
This description does not explain what is really going on. Can you give an example?

Reply:
Also, you should explain what happens if the string is not well-formed

Reply:
Sure, I will add some example to explain.